ui/components/requirements/requirement.jsx

```diff
@@ -15,13 +49,41 @@ export default function Requirement({ requirement }) {
       <div className="req-text col col-10">
         { reqTexts }
         <div className="clearfix mt3">
-          <div className="applies-to mr2">
-            Applies to: [not implemented]
-          </div>
-          <div className="sunset-date">
-            Sunset date by { requirement.policy.sunset || 'none' }
-          </div>
-          <div className="topics">
+          <Metadata
+            className="applies-to mr2"
+            name="Applies to"
+            value=""
+            nullValue="Applies to: [not implemented]"
```

Comment:
Mind adding `requirement.impacted_entity` here? It'll at least get that ball rolling.